### PR TITLE
chore: add rubocop config file at the root and add root rubocop rake task

### PR DESCRIPTION
**Pull request by trambui09**
Add ```.rubocop_root.yml ```file at the root level to run rubocop against the newly added link_transformer methods and tests. Additionally, added corresponding rake task to run ruobocop in the root directory.

### CODE REVIEW

File: test/link_transformer_test.rb [hunk 1]
```diff
@@ -39,42 +38,38 @@
...
-      and includes substantial interface changes. Existing code written for earlier
-      versions of this library will likely require updates to use this version.
-      See the [MIGRATION.md](MIGRATING.md) document for more information.
+        In order to use this library, you first need to go through the following steps
+        1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
+        1. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+        1. [Enable the API.](https://console.cloud.google.com/apis/library/translate.googleapis.com)
+        1. [Set up authentication](AUTHENTICATION.md)
+        ## Migrating from 2.x versions
+        The 3.0 release of the google-cloud-translate client is a significant upgrade
```
Comment (dazuma) [inline test/link_transformer_test.rb:67]: Any reason why you dropped the blank lines here?

**Review by dazuma — APPROVED** [review]

File: test/link_transformer_test.rb (hunk 1 above)
Comment (trambui09) [reply test/link_transformer_test.rb:67]: I don't think Rubocop was too happy with the blank lines here

Comment (dazuma) [reply test/link_transformer_test.rb:67]: We  should be able to include blank lines in a multi-line string like this. Maybe make sure the lines are truly empty (and don't have any spaces in them for example).

Comment (trambui09) [reply test/link_transformer_test.rb:67]: I was able to include the blank lines between the bullets in the list. But in other parts, I would hit a trailing whitespace error?

**Comment by trambui09** [comment]
Just added back the blank lines in the heredoc of the tests

**Comment by dazuma** [comment]
Looks great, thanks! Merging.

**Pull request merged**